### ISSUE TITLE
[APIM-Logs] Transformation of Content-Length and Transfer-Encoding header into content-length and transfer-encoding.

### ISSUE THREAD

**Issue by marijatimovska** [gravitee-io/issues#7372]
## :collision: Describe the bug
When we send a POST request vie through Postman or cURL and later go to the Logs of the API we will see that the content-length header is with a lower case instead being Content-Length. Issue is also present with transfer-encoding.

![image](https://user-images.githubusercontent.com/57534152/160142641-c57c6518-b329-4c69-9cdb-7edea30c8299.png)

## :sunrise_over_mountains: To Reproduce
1. Create/Import simple API. As an example you can put webhooks URL as an Endpoint.
2. Use Postman or cURL to send a POST request with test content. In Postman you can see that Content-Length is at is should be.
3. Go to the Logs of the API and look for Content-Length header.
4. See the error. Content-Length will be content-length.

## :rainbow: Expected behavior
Content-Length header should be with Uppercase letters like all the other headers because it can cause issue if the lowercase header is not recognized. 

## :movie_camera: Useful information
Screenshot, video, logs, other supporting material

## :computer: Desktop:

***Please complete the following information:***
 - Environment: Docker-Desktop
 - OS: Windows 11
 - Browser: Google Chrome

## :warning: Potential impacts

***What are the impacted versions?***
- 3.10.11, 3.16.0, 3.16.1, 3.15.5 and 3.15.6

**Comment by brasseld** [comment]
As per the RFC https://www.w3.org/Protocols/rfc2616/rfc2616.html

```
Each header field consists of a name followed by a colon (":") and the field value. Field names are case-insensitive.
```

**Comment by apoddany** [comment]
This behavior appears to be specific to Http2 usage and should not be present in Http1.1 except if a reverse proxy is used in between

**Comment by ThibaudAV** [comment]
There doesn't seem to be anything broken on it except for a not so pretty side in the ui. 
The change of behavior is probably due to a change of version of vertx or netty or others
Maybe it will change with the next version of vertx. I tried to find out where this was happening and if there was an issue linked, but I couldn't find it. 
And as it is not a real functional problem I did not take too much time either 
I wonder if it is not during the verification of this header that it transforms it. but as it always respects the http1 spec it is Valid

On the Gravitee side, we could add a formatter on the UI side, but I'm not sure it's desirable because it modifies reality 🤷‍♂️ 
WDYT @mouligno ?
